.gitignore

```diff
@@ -6,3 +6,4 @@ build/*
 **/*.egg-info
 **/*.idea/*
 **/.ipynb_checkpoints/*
+minervachem/mcts/ridge_pipeline.pkl
```

Comment:
Let's just change this to **/*.pkl so any pickle file will be ignored by default